Dockerfile

```diff
@@ -48,8 +48,8 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
     && apt-get update -qq \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq postgresql-client-12
 
-# Install Google Chrome for browser tests
-RUN curl -sSL https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /tmp/chrome.deb \
+# Install Google Chrome for browser tests with values {"chrome_version": "126.0.6478.182-1"}
```

Comment:
Can you add a comment with the URL where we can find the chrome version used in Odoo's runbot?

Reply:
The version in the comment above is not aligned with the code below. Let's remove it from the comment.

Reply:
:point_up:

Reply:
Done